### PR TITLE
ipn: add Funnel port check from nodeAttr

### PR DESCRIPTION
**Pull request by maisem**

### CODE REVIEW

File: tailcfg/tailcfg.go [hunk 1]
```diff
@@ -1058,6 +1058,11 @@ type PortRange struct {
 	Last  uint16
 }
 
+// Contains returns whether port is in pr.
```
Comment (bradfitz) [inline tailcfg/tailcfg.go:1061]: Reports

**Review by bradfitz — APPROVED** [review]

File: tailcfg/tailcfg.go (hunk 1 above)
Comment (maisem) [reply tailcfg/tailcfg.go:1061]: not applicable here, not a bool return.

Comment (bradfitz) [reply tailcfg/tailcfg.go:1061]: Yeah it is

Comment (maisem) [reply tailcfg/tailcfg.go:1061]: oops, i thought this was in a different place. my bad

**Review by crawshaw — APPROVED** [review]

File: tailcfg/tailcfg.go (hunk 1 above)
Comment (maisem) [reply tailcfg/tailcfg.go]: fixed

**Pull request merged**